bin/lib/recreate_database.py

```diff
@@ -14,11 +16,11 @@ def main():
     print '****************************************************'
     print
     while True:
-        response = raw_input('Proceed [y/n]? ')
-        if response.lower() == 'y':
+        response = raw_input('Proceed [yes/no]? ')
```

Comment:
I agree, but I don't think making them type out the full word is going to offer any more security if they're already determined enough to run a script called `recreate_database.py` and respond to the warning in the affirmative.